src/ralph/virtual/models.py

```diff
@@ -35,11 +35,26 @@
 logger = logging.getLogger(__name__)
 
 
+class CloudSyncProcessor(models.Model, AdminAbsoluteUrlMixin):
```

Comment:
shouldn't `AdminAbsoluteUrlMixin`be inherited first?

Reply:
Done